lisp/transient.el

```diff
@@ -1989,20 +1989,17 @@ value.  Otherwise return CHILDREN as is."
       (setq this-command 'transient-set-level))
      (t
       (setq transient--exitp nil)
-      (when (eq (if-let ((fn (transient--get-predicate-for
-                              this-original-command)))
-                    (let ((action (funcall fn)))
-                      (when (eq action transient--exit)
-                        (setq transient--exitp (or transient--exitp t)))
-                      action)
-                  (if (let ((keys (this-command-keys-vector)))
-                        (eq (aref keys (1- (length keys))) ?\C-g))
-                      (setq this-command 'transient-noop)
-                    (unless (transient--edebug-command-p)
-                      (setq this-command 'transient-undefined)))
-                  transient--stay)
-                transient--exit)
-        (transient--pre-exit))))))
+      (if-let ((fn (transient--get-predicate-for this-original-command)))
+          (when (eq transient--exit (funcall fn))
+            (setq transient--exitp (or transient--exitp t))
+            (transient--pre-exit))
+        (if (let ((keys (this-command-keys-vector)))
+              (eq (aref keys (1- (length keys))) ?\C-g))
+            (setq this-command 'transient-noop)
+          (unless (transient--edebug-command-p)
+            (setq this-command 'transient-undefined)))
+        (when (eq transient--exit transient--stay)
```

Comment:
You are comparing two constants. The result is always `nil`.

Reply:
Word.